### ISSUE TITLE
Review if there is a DICOM defacer that exists somewhere

### ISSUE THREAD

**Issue by cmadjar** [aces/Loris-MRI#624]
Check next week if can find some DICOM defacer that could be run on Windows and other platforms.

**Comment by christinerogers** [comment]
Sent Cecile my notes (spreadsheet) from research Laetitia and I have done.  
(Not linking it here because this is a public issue.)

**Comment by cmadjar** [comment]
@christinerogers I unfortunately did not find anything more promising... I added comments in the spreadsheet.

Unfortunately, PyDICOM only manipulates the header and the annotation that could have been made on a specific pixel as far as I understand (PyDICOM is what is behind DICAT BTW).

The only actual DICOM defacer I saw in the list is BrainVoyager but I think it requires a license and I doubt it is open source...

Another option would be maybe to have a tool that reads the DICOM but spits out a BIDS dataset with defaced data but we lose so much information in doing that... 

I am also very intrigued by mri_reface :-). I guess this is the tool Bosch mentioned during earlier calls?

I'll keep digging but the work you guys did is an awesome review of what is there. 👍
